ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -91,6 +95,22 @@ common_packages:
 - podman
 - tree
 
+# Packages to install for RHEL 8
+common_packages_el8:
```

Comment:
I think these are already defaults in `common` role now. Removing them from here would be one less thing to maintain. Same with the RHEL 7 ones.

Reply:
Right. Couldn't do that in my branch - I don't have your changes merged yet. Subsequent PR will remove those two common_packages...